test/unit/tuple_uint32_overflow.c

```diff
@@ -0,0 +1,74 @@
+#include "tuple.h"
+#include "unit.h"
+#include <stdio.h>
+#include <memory.h>
+#include <fiber.h>
+
+
+static char data[500];
+
+struct tuple *
+tuple_new_data(){
```

Comment:
Place bracket to the new line (according to codestyle)

Reply:
Fix